### PR TITLE
feat: Added UI Of Vedio Listing Page

### PR DESCRIPTION
**Pull request by Asuraking1n**
1- Added Vedio Cards.
2- Added Vedio Listing page.

### CODE REVIEW

File: src/components/Navbar/navbar.css [hunk 1]
```diff
@@ -21,6 +21,10 @@
     justify-content: flex-end;
     cursor: pointer;
 }
+.navLink{
+    text-decoration: none;
+    color: #fff;
```
Comment (sakshi006) [inline src/components/Navbar/navbar.css:26]: Can use css variables.

**Review by sakshi006 — APPROVED** [review]
I liked the design. The animations are very smooth and look cool.
One thing, instead of hard-coding colors, use CSS variables.
Looks good.
Clean code and file structure.

File: src/components/Navbar/navbar.css (hunk 1 above)
Comment (Asuraking1n) [reply src/components/Navbar/navbar.css:26]: I will do it after completing the whole ui

**Review by Divya70 — APPROVED** [review]
Amazing UI , i like hover effect on video

**Pull request merged**